jetty-documentation/src/main/asciidoc/administration/logging/jetty-server-dump.adoc

```diff
@@ -0,0 +1,189 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2020 Mort Bay Consulting Pty Ltd and others.
+//  ========================================================================
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+[[jetty-server-dump]]
+=== Jetty Server Dump
+
+The dump feature in Jetty provides a good snapshot of the status of the threadpool, select sets, classloaders, and so forth. 
+To get maximum detail from the dump, you need to `setDetailDump(true)` on any `QueuedThreadPools` you are using. 
+You can do this by a direct call if you are embedding Jetty, or by using properties for standalone Jetty.
```

Comment:
This is a bit too detailed to be in the introduction.   It should be in the section below.  Perhaps say what extra detail is provided and why you might want it turned off (verbosity)